config/networks.ts

```diff
@@ -25,6 +25,7 @@ const localhost: NetworkUserConfig = {
   ...(typeof MNEMONIC === 'string' && {
     accounts: { mnemonic: MNEMONIC },
   }),
+  allowUnlimitedContractSize: true,
```

Comment:
is this still needed? enabling this could prevent us from knowing whether a contract is deployable to mainnet

Reply:
Nope, will rm